Riot/Modules/Room/Views/Activities/RoomActivitiesView.m

```diff
@@ -118,64 +118,51 @@ -(void)customizeViewRendering
     {
         self.messageLabel.textColor = ThemeService.shared.theme.textSecondaryColor;
     }
+    
+    [self.resendButton.layer setCornerRadius:5];
+    self.resendButton.clipsToBounds = YES;
+    [self.resendButton setTitle:NSLocalizedStringFromTable(@"retry", @"Vector", nil) forState:UIControlStateNormal];
```

Comment:
Do we need a text here? 
It will not fit well with most languages. Example with French:

<img width="359" alt="Screenshot 2021-03-10 at 15 52 21" src="https://user-images.githubusercontent.com/8418515/110654859-cee7d000-81be-11eb-8ac8-129fccb1825c.png">

Reply:
Fixed  by updated compression priority

<img width="426" alt="Screenshot 2021-03-10 at 22 04 37" src="https://user-images.githubusercontent.com/15386762/110698030-6c59f880-81ed-11eb-89d2-00825eb34c2d.png">